### PR TITLE
cephdeploy.conf: use more meaningful example apt repo url

### PR DESCRIPTION
**Pull request by zhouyuan**
Signed-off-by: Yuan Zhou <yuan.zhou@intel.com>

### CODE REVIEW

File: ceph_deploy/conf/cephdeploy.py [hunk 1]
```diff
@@ -43,14 +43,14 @@
 
 # apt repos:
 # [myrepo]
-# baseurl = https://user:pass@example.org/
+# baseurl = https://user:pass@example.org/ceph-deb-trusty-x86_64-basic/ref/hammer
```
Comment (alfredodeza) [inline ceph_deploy/conf/cephdeploy.py]: maybe a public url would make sense here (vs. the example.org one)

File: ceph_deploy/conf/cephdeploy.py [hunk 2]
```diff
@@ -28,29 +28,29 @@
 
 # yum repos:
 # [myrepo]
-# baseurl = https://user:pass@example.org/rhel6
-# gpgurl = https://example.org/keys/release.asc
+# baseurl = http://gitbuilder.ceph.com/ceph-rpm-centos7-x86_64-basic/ref/hammer
+# gpgurl = https://ceph.com/git/?p=ceph.git;a=blob_plain;f=keys/autobuild.asc
```
Comment (trhoden) [inline ceph_deploy/conf/cephdeploy.py]: switch from ceph.com/git to git.ceph.com

File: ceph_deploy/conf/cephdeploy.py [hunk 3]
```diff
@@ -28,29 +28,29 @@
...
 # default = True
 # extra-repos = cephrepo  # will install the cephrepo file too
 #
 # [cephrepo]
 # name=ceph repo noarch packages
-# baseurl=http://ceph.com/rpm-emperor/el6/noarch
+# baseurl=http://ceph.com/rpm-hammer/el6/noarch
```
Comment (trhoden) [inline ceph_deploy/conf/cephdeploy.py]: these should now be `download.ceph.com`

File: ceph_deploy/conf/cephdeploy.py [hunk 4]
```diff
@@ -28,29 +28,29 @@
...
 # enabled=1
 # gpgcheck=1
 # type=rpm-md
-# gpgkey={gpgurl}
+# gpgkey=https://ceph.com/git/?p=ceph.git;a=blob_plain;f=keys/release.asc
```
Comment (trhoden) [inline ceph_deploy/conf/cephdeploy.py]: git.ceph

File: ceph_deploy/conf/cephdeploy.py [hunk 5]
```diff
@@ -28,29 +28,29 @@
...
 
 # apt repos:
 # [myrepo]
-# baseurl = https://user:pass@example.org/
-# gpgurl = https://example.org/keys/release.asc
+# baseurl = http://gitbuilder.ceph.com/ceph-deb-trusty-x86_64-basic/ref/hammer
+# gpgurl = https://ceph.com/git/?p=ceph.git;a=blob_plain;f=keys/autobuild.asc
```
Comment (trhoden) [inline ceph_deploy/conf/cephdeploy.py]: git.ceph

File: ceph_deploy/conf/cephdeploy.py [hunk 6]
```diff
@@ -28,29 +28,29 @@
...
 # default = True
 # extra-repos = cephrepo  # will install the cephrepo file too
 #
 # [cephrepo]
-# baseurl=http://ceph.com/rpm-emperor/el6/noarch
-# gpgkey={gpgurl}
+# baseurl=http://ceph.com/debian-hammer
```
Comment (trhoden) [inline ceph_deploy/conf/cephdeploy.py]: download.ceph

**Pull request merged**